cirq-core/cirq/ops/measure_util.py

```diff
@@ -81,8 +81,29 @@ def measure_paulistring_terms(
     return [PauliMeasurementGate([pauli_basis[q]], key=key_func(q)).on(q) for q in pauli_basis]
 
 
+# pylint: disable=function-redefined
+
+
+@overload
+def measure(
+    *target: raw_types.Qid,
+    key: Optional[Union[str, 'cirq.MeasurementKey']] = None,
+    invert_mask: Tuple[bool, ...] = (),
+) -> raw_types.Operation:
+    pass
+
+
+@overload
+def measure(
+    *target: Iterable[raw_types.Qid],
+    key: Optional[Union[str, 'cirq.MeasurementKey']] = None,
+    invert_mask: Tuple[bool, ...] = (),
+) -> raw_types.Operation:
```

Comment:
nit: Could modify this slightly to accept exactly one arg:
```suggestion
def measure(
    __target: Iterable[raw_types.Qid],
    *,
    key: Optional[Union[str, 'cirq.MeasurementKey']] = None,
    invert_mask: Tuple[bool, ...] = (),
) -> raw_types.Operation:
```
(The `__` prefix on the arg name indicates to mypy that this is a positional-only arg for which we don't care about the name.)